testdata/full_compute_instance.json

```diff
@@ -135,7 +135,8 @@
             "bar",
             "foo"
           ]
-        }
+        },
+        "zone": "projects/test-project/zones/us-central1-a"
```

Comment:
This file is actually generated by tfv via tfplan -> json. I find another issue where the tfv does not generate this field... I wonder why the CAI does not provide the zone field...

Reply:
😟